sormas-app/app/src/main/res/values/strings.xml

```diff
@@ -143,6 +143,7 @@
     <string name="caption_case_tasks">Case Tasks</string>
     <string name="caption_case_events" >Case Events</string>
     <string name="caption_case_immunizations" >Case Immunizations</string>
+    <string name="caption_case_vaccinations" >Case Vaccinations</string>
```

Comment:
Please remove the whitespace before the closing bracket (also for contact vaccinations below)

Reply:
I removed white spaces in all places from strings.xml